druid/src/core.rs

```diff
@@ -215,9 +243,12 @@ impl<T, W: Widget<T>> WidgetPod<T, W> {
         self.layout_rect()
     }
 
-    /// The layout rectangle.
+    /// Returns the layout [`Rect`].
+    ///
+    /// This will be same value as set by [`set_layout_rect`].
```

Comment:
Should this be something like "This will be *the* same value as set by `set_layout_rect`"?